contrib/dropwizard5/src/test/java/io/opencensus/contrib/dropwizard5/DropWizardMetricsTest.java

```diff
@@ -0,0 +1,373 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.dropwizard5;
+
+import static com.google.common.truth.Truth.assertThat;
+import static io.opencensus.contrib.dropwizard5.DropWizardMetrics.DEFAULT_UNIT;
+import static io.opencensus.contrib.dropwizard5.DropWizardMetrics.NS_UNIT;
+
+import io.dropwizard.metrics5.Counter;
+import io.dropwizard.metrics5.Gauge;
+import io.dropwizard.metrics5.Histogram;
+import io.dropwizard.metrics5.Meter;
+import io.dropwizard.metrics5.MetricFilter;
+import io.dropwizard.metrics5.MetricName;
+import io.dropwizard.metrics5.Timer;
+import io.opencensus.common.Timestamp;
+import io.opencensus.metrics.LabelKey;
+import io.opencensus.metrics.LabelValue;
+import io.opencensus.metrics.export.Metric;
+import io.opencensus.metrics.export.MetricDescriptor;
+import io.opencensus.metrics.export.MetricDescriptor.Type;
+import io.opencensus.metrics.export.Summary;
+import io.opencensus.metrics.export.Summary.Snapshot;
+import io.opencensus.metrics.export.Summary.Snapshot.ValueAtPercentile;
+import io.opencensus.metrics.export.Value;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link DropWizardMetrics}. */
+@RunWith(JUnit4.class)
+public class DropWizardMetricsTest {
+
+  private io.dropwizard.metrics5.MetricRegistry metricRegistry;
+  private DropWizardMetrics dropWizardMetrics;
+
+  @Before
+  public void setUp() {
+    metricRegistry = new io.dropwizard.metrics5.MetricRegistry();
+    dropWizardMetrics = new DropWizardMetrics(Collections.singletonList(metricRegistry));
+  }
+
+  @Test
+  public void collect() throws InterruptedException {
```

Comment:
Instead of one giant test, can you please split test by ```collect_Counter```, ```collect_Gauge```, ```collect_Meter``` etc. Please refer https://github.com/census-instrumentation/opencensus-java/blob/master/contrib/dropwizard/src/test/java/io/opencensus/contrib/dropwizard/DropWizardMetricsTest.java

Reply:
I suggest we do it in a subsequent patch, I saw the split in the dropwizard old package but for time being I'd keep it like it